packages/react-instantsearch-core/src/connectors/__tests__/connectSortBy.js

```diff
@@ -58,18 +66,26 @@ describe('connectSortBy', () => {
     });
 
     it('refines the index parameter', () => {
-      params = getSP(new SearchParameters(), {}, { sortBy: 'yep' });
+      params = connect.getSearchParameters.call(
+        {},
+        new SearchParameters(),
+        { contextValue },
+        { sortBy: 'yep' }
+      );
       expect(params.index).toBe('yep');
     });
 
     it('registers its id in metadata', () => {
-      const metadata = getMetadata({});
+      const metadata = connect.getMetadata.call({}, { contextValue });
```

Comment:
It's useless in that case, we don't use the context value inside the `getMetadata`.